src/rpc/server.cpp

```diff
@@ -365,6 +365,22 @@ void JSONRPCRequest::parse(const UniValue& valRequest)
         params = UniValue(UniValue::VARR);
     else
         throw JSONRPCError(RPC_INVALID_REQUEST, "Params must be an array or object");
+
+    // Log arguments unless sensitive.
+    static const std::set<std::string> blacklist = {
+        "encryptwallet",
+        "walletpassphrase",
+        "walletpassphrasechange",
+    };
```

Comment:
Looks like this list is incomplete. Would be nice if it can be deduplicated with the existing `QStringList historyFilter`

Reply:
This blacklist should be pulled out of a config rather than buried in the code.

Reply:
What do you mean by "config"? Are you referring to `~/.bitcoin/bitcoin.conf`?

Reply:
I mean that it should go into a common module, probably `./src/util/` or similar.